plugins/woocommerce-admin/client/tasks/fills/components/load-sample-product-modal.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+import { Spinner } from '@wordpress/components/build/ui';
```

Comment:
We have three different Spinner components:

- [`@woocommerce/components/spinner`](https://woocommerce.github.io/woocommerce-admin/#/components/)
- [`@wordpress/components/spinner`](https://wordpress.github.io/gutenberg/?path=/story/components-spinner--default)
- [`@wordpress/components/ui/spinner`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-spinner--default)

And we need to use the one from `@wordpress/components/ui/spinner`, but it's not included in external wp.components.


`import { Spinner } from '@wordpress/components/ui';` <- This doesn't work because ui doesn't export in @wordpress/components's index.js